src/grandorgue/GrandOrgueFrame.cpp

```diff
@@ -205,6 +204,8 @@ GOrgueFrame::GOrgueFrame(
   m_file_menu->Append(ID_FILE_IMPORT_COMBINATIONS, _("Import &Combinations"), wxEmptyString, wxITEM_NORMAL);
   m_file_menu->Append(ID_FILE_EXPORT, _("&Export Settings/Combinations"), wxEmptyString, wxITEM_NORMAL);
   m_file_menu->AppendSeparator();
+  m_file_menu->Append(ID_SETTINGS, wxT("&Settings..."), wxEmptyString, wxITEM_NORMAL);
```

Comment:
@larspalo Usually, it's better not to mix concepts in a same PR. So, if you want to change things that are not linked together, it's better to do separate PR's.